app/views/external_users/claims/basic_fees/_additional_fee_fields.html.haml

```diff
@@ -44,13 +44,8 @@
       = f.hidden_field :price_calculated, value: f.object.price_calculated?
 
     .fee-calc-help-wrapper.form-group.hidden
-      %details
-        %summary
-          %span.summary
-            = t('.help_how_we_calculate_rate_title')
-        .panel.panel-border-narrow
-          %p
-            = t('.help_how_we_calculate_rate_body')
+      = govuk_detail t('.help_how_we_calculate_rate_title'), false do
```

Comment:
`false` is default so i would do without the second arg here.

unless "openness" is dynamic (i.e the result of a method call) or non-default  i would not add it - which it is not in this case and all cases below where it is hardcoded as false

In context, i only added the  `open` arg option for those situatiosn where open needs to be decided by code.I defaulted it to false as that is the typical use. So i would only add a second arg at all if it references a some kind of boolean metthod.

Reply:
as and aisde, and not a blocker, you could do this in a one-liner i think - your preference entirely tho!
```
 = govuk_detail t('.help_how_we_calculate_rate_title')  { t('.help_how_we_calculate_rate_body') }
```

Reply:
> as and aisde, and not a blocker, you could do this in a one-liner i think - your preference entirely tho!
> 
> ```
>  = govuk_detail t('.help_how_we_calculate_rate_title')  { t('.help_how_we_calculate_rate_body') }
> ```

One-liner fails, unable to resolve at present therefore continuing with the block approach.